### PR TITLE
meshwizard: remove local out of function

### PR DESCRIPTION
**Pull request by FreifunkUFO**

### CODE REVIEW

File: contrib/package/meshwizard/files/usr/bin/meshwizard/helpers/setup_network.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ if [ "$netrenamed" = "lan" ]; then
 fi
 
 # Setup IPv6 for the interface
-local ip6addr
+ip6addr
```
Comment (hnyman) [inline contrib/package/meshwizard/files/usr/bin/meshwizard/helpers/setup_network.sh]: This looks suspicious. There is no value assignment, so the line should be removed wholly, I think. Plain "ip6addr" sounds like a command...

Comment (FreifunkUFO) [inline contrib/package/meshwizard/files/usr/bin/meshwizard/helpers/setup_network.sh]: thanks, yes it should be deleted

**Comment by FreifunkUFO** [comment]
Please who wants (and is able) to help us with this?

**Comment by blogic** [comment]
@FreifunkUFO properly annotate your commit and explain what it does and why. i am sure people will help once you did so

**Comment by FreifunkUFO** [comment]
for explanation: there are some similiar tickets on other packages, f.i. jow wrote as a description:
Local variable declarations outside of functions are illegal since the Busybox
update to v1.25.0, therfore remove them from the appropriate places.

here https://bugs.lede-project.org/index.php?do=details&task_id=188&opened=171&status[0]=
https://git.lede-project.org/?p=source.git;a=commitdiff;h=1c09849f6cf1b8bd94819ca11910ce444c3d3019;hp=df9efc94970ccc1e43915839ec3f49d54729cc3d
https://git.lede-project.org/?p=source.git;a=commitdiff;h=864b2d113ad0815c2444575014e7cf51498944db

**Comment by hnyman** [comment]
Yes, the underlying reason is clear, but you have not removed the one extra "ip6addr" line that I pointed out (and you agreed in October that it should be deleted), and you have not edited your commit message to be properly signed-off-by. 

You proposed a change, I pointed out that it is partially wrong, and now we are waiting for you to fix your commit.

**Comment by FreifunkUFO** [comment]
yes, one line has to be removed and a signed-off-by.
im waiting for someone who can help me with this, or someone who is able to do that by his/her own. its too tricky for me.

**Comment by hnyman** [comment]
closed via 5852affbbc286e7d11218d990f8ecba0f525b91b

But please learn to edit your PRs. Rather simple and easy to follow advice can be found in
https://github.com/openwrt/luci/blob/master/CONTRIBUTING.md#advice-on-pull-requests

And please remember to sign-off all your commits with your real name in future.

**Pull request closed without merging**